### PR TITLE
Add Numa aliment test for DPDK workload

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: functests/dpdk/dpdk.go [hunk 1]
```diff
@@ -98,3 +131,70 @@ func findDPDKWorkloadPod() *corev1.Pod {
...
+	}
+
+	return numaNode, nil
+}
+
+// findNUMAForSRIOV finds the NUMA node for a give PCI address
+func findNUMAForSRIOV(pod *corev1.Pod) (int, error) {
+	buff, err := pods.ExecCommand(client.Client, *pod, []string{"env"})
+	Expect(err).ToNot(HaveOccurred())
+	pciAddres := ""
```
Comment (fedepaol) [inline functests/dpdk/dpdk.go]: nit: pciAddress

Comment (SchSeba) [reply functests/dpdk/dpdk.go]: Done thanks!

**Pull request merged**